content/en/docs/overview/singleton-sdk/python-client-introduction.md

```diff
@@ -1,8 +1,202 @@
+
+
+# APIs Available
+* [Factory Class](#Factory-Class)
+   * [I18n](#I18n)
+* [Interface For Release](#Interface-For-Release)
+   * [Release](#Release)
+* [Interface For Configuration](#Interface-For-Configuration)
+   * [Config](#Config)
+* [Interface For Translation](#Interface-For-Translation)
+   * [Translation](#Translation)
+
+## Factory Class
+### I18n
+
+* It's the factory class that creates and initializes the configuration object and the release object.
+
+```python
+class I18n():
+    def add_config_file(cls, config_file)
+    def add_config(cls, base_path, config_data)
+    def set_current_locale(cls, locale)
+    def get_current_locale(cls)
+    def get_release(cls, product, version)
+```
+
+### I18n / add_config_file
+
+* Load a configuration json or yaml file and initialize a correspondent release object.
+
+| Parameter | Type | Description |
+| ------ | ------ | ------ |
+| config_file | string | Configuration file |
+
+### I18n / add_config
+
+* Load a configuration json or yaml text with a base directory and initialize a correspondent release object.
+
+| Parameter | Type | Description |
+| ------ | ------ | ------ |
+| base_path | string | Base directory |
+| config_data | string | Configuration text in json or yaml |
+
+### I18n / set_current_locale
+
+* Set locale of current thread.
+
+| Parameter | Type | Description |
+| ------ | ------ | ------ |
+| locale | string | Locale name |
+
+### I18n / get_current_locale
+
+* Get locale of current thread.
+
+| Return | Description |
+| ------ | ------ |
+| string | Locale name |
+
+### I18n / get_release
+
+| Parameter | Type | Description |
+| ------ | ------ | ------ |
+| product | string | Product name |
+| version | string | Version |
+
+* Get release object.
+
+| Return | Description |
+| ------ | ------ |
+| [Release](#Release) | Release object |
+
+
+## Interface For Release
+### Release
+```python
+class Release:
+    def get_config(self)
+    def get_translation(self)
+```
+
+### Release / get_config
+
+* Get the configuration object owned by the release object.
+
+| Return | Description |
+| ------ | ------ |
+| [Config](#Config) | The configuration object |
+
+### Release / get_translation
+
+* Get the translation object owned by the release object.
+
+| Return | Description |
+| ------ | ------ |
+| [Translation](#Translation) | The translation object |
+
+
+## Interface For Configuration
+### Config
+```python
+class Config:
+    def get_config_data(self)
+    def get_info(self)
+```
+### Config / get_config_data
+
+* Get configuration data.
+
+| Return | Description |
+| ------ | ------ |
+| object | Configuration data |
+
+### Config / get_info
+
+* Get brief information of configuration.
+
+| Return | Description |
+| ------ | ------ |
+| object | Brief information data |
+
+
+## Interface For Translation
+### Translation
+```python
+class Translation:
+    def get_string(self, component, key, **kwargs)
```

Comment:
```suggestion
    def get_string(self, component, key, **args)
```

Reply:
kwargs is a python style name that has fixed meaning.